utils/data.tsx

```diff
@@ -1575,9 +1575,9 @@ export const BoardNavData = {
 
 export const DescribeCardsData = [
   {
-    title: 'Paloma AI LighNodes',
+    title: 'Paloma AI LightNodes',
     describe:
-      'The Paloma LightNode is a blockchain program that secures the Paloma network by delegating stake to top cross-chain validators. Participants, called pigeons, earn GRAIN rewards as their clients work continuously to ensure network security.',
+      'The Paloma AI LightNode is a lightweight Paloam client that both secures the Paloma network by delegating stake to top cross-chain validators as well as stores your AI context windows. Paloma AI LightNodes, called pigeons, earn increasing GRAIN rewards as more tokens and AI agents are launched on Paloma across multiple blockchains.',
```

Comment:
@jamie088 spelling mistake. a lightweight Paloam needs to be a lightweight Paloma